values.schema.json

```diff
@@ -18,7 +18,143 @@
                       "additionalProperties": false,
                       "properties": {
                         "airflow": {
-                          "type": "object"
+                          "allOf": [
+                            {"$ref": "https://raw.githubusercontent.com/apache/airflow/main/chart/values.schema.json"},
```

Comment:
@jedcunningham I dislike that I'm pointing to main here as that changes frequently and I'd like to lock this to a specific releases json schema

Reply:
Can we ref a tag on a release branch instead?

Reply:
Would this link be the one to use? https://raw.githubusercontent.com/apache/airflow/2.2.1/chart/values.schema.json

Whats our risk of someone on OSS side changing the tag commit.

Reply:
This is the one you want:
https://github.com/apache/airflow/blob/helm-chart/1.3.0/chart/values.schema.json

Reply:
The tag shouldn't change.